guides/v2.3/extension-dev-guide/routing.md

```diff
@@ -116,7 +116,7 @@ Where:
 
 * `%routerId` - specifies the name of the router in Magento.
     See the reference tables in the [Router class section].
-* `%routeId%` - specifies the unique node id for this route in Magento.
+* `%routeId%` - specifies the unique node id for this route in Magento, is also the first segment of its associated layout handle XML filename (`routeId_controller_action.xml`).
```

Comment:
Hi @drpayyne. Thank you for your collaboration. Since the same description is fair for Magento 2.2, could you add this description to the corresponding file for 2.2 as well, please? 
Thank you

Reply:
Done, thank you!